components/EventsFallFest/EventsFallFestUniversityDirectory.vue

```diff
@@ -43,6 +43,7 @@
 
 <script setup lang="ts">
 import { TextLink } from "~/types/links";
+import { sortEvents } from "~/utils/event-schemaorg";
```

Comment:
Exports in the `utils` directory are globally available, so we don't need to import it.

With that in mind, I believe that it's a good idea to have very descriptive names for these functions and files.

Since this particular function sorts all kind of events, I think that it's indeed a good idea to call it `sortEvents`, but I also think that it should be in its own file (something like **utils/sortEvents.ts** and not included in a file related to the Schema.org stuff.

Reply:
Thanks! I think is a good idea. Changes applied!